packages/loader/container-loader/src/containerStorageAdapter.ts

```diff
@@ -52,6 +52,8 @@ export class ContainerStorageAdapter implements IDocumentStorageService, IDispos
 		return this._summarizeProtocolTree === true;
 	}
 
+	public loadingGroupIds: Record<string, ISnapshot> = {};
```

Comment:
Name does not seem to reflect what is stored - snapshots.
Maybe partialSnapshots?